docker4bots/00_setup_shm_for_test.sh

```diff
@@ -1,8 +1,13 @@
 #!/bin/bash -e
 
+if [ '$EUID' -ne 0 ]
+    then echo 'This script needs to be run as root.'
+    exit
+fi
+
 source config.sh
 
-sudo mount -t tmpfs -o noexec,size=1G none $SPN_SHM_HOSTDIR/
-sudo chown $(id -u):$(id -g) $SPN_SHM_HOSTDIR
+mkdir -p $SPN_SHM_HOSTDIR/
+mount -t tmpfs -o noexec,size=1G none $SPN_SHM_HOSTDIR/
+chown $(id -u):$(id -g) $SPN_SHM_HOSTDIR
```

Comment:
I'm not sure if this still works when the whole script runs as root. The `$SPN_SHM_HOSTDIR` needs to belong to (or at least be writable) by the user running the gameserver (which is not root, I hope ;-) ).